docs/Pflichtenheft.tex

```diff
@@ -257,7 +259,7 @@ \section{Produktdaten}
  \item[D50] Die \gls{Uberwachungskonsole} speichert, welche Verläufe zuletzt angezeigt wurden.
  \item[D110] Die Fertigungssimulation speichert die zuletzt genutzten Ports der OPC UA Server.
  \item[D120] Die Fertigungssimulation speichert den zuletzt eingestellten \gls{Jitter}.
- \item[D130] Die Fertigungssimulation speichert \emph{eventuell (sofern implementiert)} aufgenommene \glspl{Makro}.
+ \item[D130] Die Fertigungssimulation speichert \glspl{Simulations-Szenario} in einzelnen Dateien (sofern implementiert).
 \end{enumerate}
```

Comment:
In optionale Funktionalitaet muss "(sofern implementiert)" nicht erwaehnt werden.